examples/history EETS .txt

```diff
@@ -0,0 +1,87 @@
+As its name states, EETS was begun as a 'club', and it retains certain features of that even now. 
```

Comment:
Using spaces in file names - bad idea, especially at the end of the name.
I prefer to avoid this additional folder and rename the file to something similar to `testing_data.txt`, etc.

Reply:
I agree, done